File: DSA/YashKrishanGupta_590014798/Day_30_Question_3.java
import java.util.Scanner;
public class Day_30_Question_3 {
    // Solution 3
    public static int countSetBits(int n){
        int count = 0;
        while(n>0){
            if((n&1)==1) count++;
            n>>=1;
        }
        return count;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number : ");
        int num = sc.nextInt();
        int count = countSetBits(num);
        System.out.print("Count of set bits : ");
        System.out.print(count);
    }
}

File: DSA/Ayush_Mishra_590014844/Day_30_Question_3.c
#include <stdio.h>

int main() {
    int n, count = 0;
    printf("Enter the number (n): ");
    scanf("%d", &n);
    while (n) {
        n = n & (n - 1);
        count++;
    }
    printf("Number of set bits: %d\n", count);
    return 0;
}
